docs/releaseNotes.yml

```diff
@@ -40,7 +40,13 @@ items:
         type: security
         body: >-
           Updated Golang to 1.19.2 to address the CVEs: CVE-2022-2879, CVE-2022-2880, CVE-2022-41715.
-
+      - title: Fixed finding ingress resource tls secrets
+        type: bugfix
+        body: >-
+          Previous versions of $productName$ required a workaround using <code>TLSContexts</code> to find tls secrets referenced from
```

Comment:
One last comment, the real fix here is that we are now listening for both `v1beta1` and `v1` ingress resources where as before it was only looking at `v1beta1` resources,  correct?

Reply:
If so then since we have a merge conflict can we make the text here reflect that. Other than than I'm ok with the change.